db/migrate/20151013214647_set_redeemable_true_on_virtual_gift_cards.rb

```diff
@@ -0,0 +1,11 @@
+class SetRedeemableTrueOnVirtualGiftCards < ActiveRecord::Migration
+  def up
+    Spree::VirtualGiftCard.find_each do |gift_card|
+      gift_card.update_attributes(redeemable: true)
```

Comment:
Good call on the bang, updated. We can't really have a down case, because setting them all to redeemable false is not the opposite of the up, since all current gift cards -are- redeemable.